features/keyboard-map.yml

```diff
@@ -0,0 +1,6 @@
+name: Keyboard map
+description: Keyboard map allows websites to convert from a given `code` value to a valid `key` value that can be shown to the user to identify the given key. It is intended to be used by web applications that want to treat the keyboard as a set of buttons and need to describe those buttons to the user.
```

Comment:
```suggestion
description: The `navigator.keyboard.getLayoutMap()` API returns a map from key codes to human readable key names. It can be used to describe buttons to the user when using the keyboard as a set of buttons, such as in games.
```